### PR TITLE
Added Support for Chevron in UIButton

### LINKED ISSUE #4167: [Mobile][Usability] [No Cheveron on Action.ShowCards]
# Platform

- [ ] Android
- [x] iOS

# Author or host

Host on Webex Teams

# Version of SDK

We have our own fork of AdaptiveCards@v1.2.6

# Details

Action.ShowCard buttons on JS Renderer include a chevron that points down when the card is expanded and points up when the card is closed.  This helps user differentiate these types of buttons from the other types.    The mobile renderers should do something similar.

JS Render:

JS-Render

Android Render:

Android-Render

ios render:

ios-render

### PR DESCRIPTION
**Pull request by jwoo-msft**
## Related Issue
Fixed #4167 for iOS

## Description
* Added support for chevron for Action.ShowCard or customizations for buttons' different states.
* ACRButtonExpandable.xib is added.
* ACRButtonExpandable.xib is used for Action.ShowCard. All other actions types use ACRButton.xib.
* As one solution of many, one can add a chevron up image to default sate, and add a chevron down image to selected state.
* spacing between the chevron and title can be adjusted by the image's left inset in the xib.
* since iOS doesn't support text wrap in buttons, please set line break mode to "Clip", to make the horizontal scroll to work properly in case of the width overflow.
* disclaimer: ACRButtonExpandable.xib in this PR does not contain changes for actual chevron images described in the description. Please refer to [PR4221](https://github.com/microsoft/AdaptiveCards/pull/4221) for complete solution. 

## How Verified
How you verified the fix, including one or all of the following: 
1. Existing relevant unit/regression tests were run and verified.
image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/4222)

### CODE REVIEW

**Comment by shalinijoshi19** [comment]
@jwoo-msft what's our story for Android? What about the other platforms? @golddove / @RebeccaAnne

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButtonExpandable.xib [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="14490.70" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
+    <device id="retina4_7" orientation="portrait">
+        <adaptation id="fullscreen"/>
```
Comment (shalinijoshi19) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButtonExpandable.xib:4]: Curious - how does the xib approach work again? Do they optionally get to pull this in to their source/project or something like that?

**Comment by jwoo-msft** [comment]
> @jwoo-msft what's our story for Android? What about the other platforms? @golddove / @RebeccaAnne

@shalinijoshi19 UWP should be able to do this through XML. We need to investigate on Android if native styling approaches are feasible.

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButtonExpandable.xib (hunk 1 above)
Comment (jwoo-msft) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRButtonExpandable.xib:4]: This XIB will be part of iOS SDK. Devs will need to modify this XIB to have "chevrons" for showcard buttons. I've provided such changes in the XIB as part of PR #4221 for reference.

**Pull request merged**